Dockerfile

```diff
@@ -7,11 +7,15 @@ RUN cargo build --release && \
     cd / && rm -rf /source
 
 # Create runtime container
+# Note that we need a small init process for PID 1 that forwards signals.
+# See https://github.com/Yelp/dumb-init
 FROM debian:stable-slim
+RUN apt-get update && apt-get install dumb-init && rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
RUN apt-get update && apt-get upgrade && apt-get install dumb-init && rm -rf /var/lib/apt/lists/*
```

Reply:
No, upgrades are generally not intended to be done by containers. Instead, the base image will receive regular updates (which is the case for debian).

This is even mentioned in the [Docker best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run):

![2019-12-31-002111_900x87_scrot](https://user-images.githubusercontent.com/105168/71604617-7b620700-2b63-11ea-901c-9550a7a6819e.png)

Reply:
Okay, how do you make sure that you always pull the latest image?

Reply:
We auto-rebuild the image every week in CI: https://github.com/coredump-ch/status/blob/72ab45b81cca8522547247503ab7a9e9e62b74ea/.circleci/config.yml#L89-L99

Reply:
(...and then auto-deploy that updated image with watchtower :slightly_smiling_face:)

Reply:
LGTM